src/app/pages/FrontPage/index.jsx

```diff
@@ -79,6 +84,7 @@ const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
 
   return (
     <>
+      {hasAds && !isAmp && <CanonicalAdBootstrapJs />}
```

Comment:
Added additional unit tests for this, I didn't add data-e2e yet because it's a Helmet script rather than DOM element, but will keep this in mind for future updates.